### ISSUE TITLE
Travis tests for macOS fails: `ValueError: numpy.ndarray size changed, may indicate binary incompatibility`

### ISSUE THREAD

**Issue by fzeiser** [oslocyclotronlab/ompy#178]
The Travis tests for macOS (with and without libomp) fail on the current master, 70c3d66, with following error message:
``` pytb
______________________ ERROR collecting test_firstgen.py _______________________
tests/test_firstgen.py:3: in <module>
    import ompy as om
ompy/__init__.py:14: in <module>
    from ompy.rebin import *
ompy/rebin.pyx:1: in init ompy.rebin
    import numpy as np
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
[...]
```

More info:
https://travis-ci.com/github/oslocyclotronlab/ompy/jobs/479103734

and https://docs.travis-ci.com/user/reference/osx/:
> Travis CI uses macOS 10.13 and Xcode 9.4.1 by default.

I've only changed some comments in the getting-started notebook, which is not part of the tests currently. So there is some other problem. I'm not 100% convinced that this occurs for other users.

**Comment by vetlewi** [comment]
Update: I just updated numpy from v1.19.5 to v1.20.0 and everything works.

**Comment by fzeiser** [comment]
Maybe some changes / updates to some packages for macOS? Travis build/tested 05a3a50 without problems 18 days ago:
https://travis-ci.com/github/oslocyclotronlab/ompy/builds/213479922

**Comment by vetlewi** [comment]
> Maybe some changes / updates to some packages for macOS? Travis build/tested [05a3a50](https://github.com/oslocyclotronlab/ompy/commit/05a3a503866acdfa36931fbb01c9d8eb337495fc) without problems 18 days ago:
> https://travis-ci.com/github/oslocyclotronlab/ompy/builds/213479922
> 
> (and from a _brief_ inspection I see that at least `cython`, `numpy` and `scipy` ) apparently have the same version in both builds
> 
> ```
> Downloading Cython-0.29.21-cp37-cp37m-macosx_10_9_x86_64.whl (1.9 MB)
> Downloading scipy-1.6.0-cp37-cp37m-macosx_10_9_x86_64.whl (30.7 MB)
> Requirement already satisfied: numpy in /usr/local/lib/python3.7/site-packages (1.18.1)
> ```

There might have been a macOS update that broke compatibility?

For future tests, maybe add a script that outputs the versions of all packages before running the tests?
Following example of python code outputs useful diagnostics (if ran in ipython):
```python
import numpy
import cython
import pandas
import matplotlib
import pymultinest
import scipy
import uncertainties
import tqdm
import pathos
import pybind11

import watermark


%load_ext watermark
%watermark -m -u -d -v -iv -w

```

**Comment by fzeiser** [comment]
Closed through #182.
